client/homebrew/pages/userPage/brewItem/brewItem.jsx

```diff
@@ -113,6 +114,9 @@ const BrewItem = createClass({
 				<span>
 					<i className='fas fa-eye' /> {brew.views}
 				</span>
+				<span>
+					<i className='far fa-file' /> {brew.pageCount || 1}
+				</span>
```

Comment:
could this be made to not display at all if `!brew.pageCount` ?

Reply:
I've modified it so that it will only display if `pageCount>1`.

Reply:
As per request from @calculuschild, this is now set to display if `brew.pageCount` exists, even if it is only a single page.